httpbakery/client.go

```diff
@@ -199,6 +199,12 @@ func (c *Client) DischargeAll(ctx context.Context, m *bakery.Macaroon) (macaroon
 	return bakery.DischargeAllWithKey(ctx, m, c.AcquireDischarge, c.Key)
 }
 
+// DischargeAllUnbound is like DischargeAll except that it does not
```

Comment:
does this function really add value, why not rely just using bakery.Slice.DischargeAll?

Reply:
It saves the client knowing about AcquireDischarge and Key, so I *think* it's just about worth it. I added it because I wanted to use it.

Reply:
Also, if DischargeAllUnbound isn't worth it, then DischargeAll probably isn't either. But I think it is... just about.